File: 7-kyu/80's-kids-#5--you-can't-do-that-on-television/javascript/solution.js
const bucketOf = str => {
  const slimeTest = /(i don't know)|(slime)/gi.test(str);
  const waterTest = /(water)|(wet)|(wash)/gi.test(str);
  let contents = "air";
  if (slimeTest) {
    if (waterTest) {
      contents = "sludge";
    } else {
      contents = "slime";
    }
  }
  else if (waterTest) {
    contents = "water";
  }
  return contents;
};

File: 7-kyu/80's-kids-#5--you-can't-do-that-on-television/python/solution.py
def bucket_of(said):
    slime_triggers = ['i don\'t know', 'slime']
    water_triggers = ['water', 'wet', 'wash']
    
    check_drop = lambda phrases: any(phrase in said.lower() for phrase in phrases)
    
    drop_water = check_drop(water_triggers)
    drop_slime = check_drop(slime_triggers)
    
    if drop_water and drop_slime:
        return 'sludge'
    elif drop_water:
        return 'water'
    elif drop_slime:
        return 'slime'
    return 'air'
